libs/blocks/quiz/quiz.js

```diff
@@ -56,7 +56,14 @@ const App = ({
         strMap[question.q] = question;
       });
 
-      setUserFlow([questions.questions.data[0].questions]);
+      if (!!Object.keys(storedQuizState).length
+        && !!storedQuizState?.userFlow.length
+        && !!storedQuizState?.userSelection.length) {
+        setUserFlow(storedQuizState.userFlow);
+        updateUserSelection(storedQuizState.userSelection);
```

Comment:
Missing test coverage

Reply:
Thanks, @fullcolorcoder, for noting that the state wasn't cleared. I logged a story for that update. https://jira.corp.adobe.com/browse/MWPW-143761

Reply:
And on reflection, I moved my story to a sub-task of the current story. We may as well make the changes before merging this one.

Reply:
@fullcolorcoder This should now be cleared as soon as handleOnNextClick() is called - when the next button is clicked for the first time after loading a stored quiz state

Reply:
Also, regarding test coverage, parts of this will change when we bring in the ML field. I want to have a complete picture before I add the tests.